### PR TITLE
Bip only offer  free 7 days trial

### PR DESCRIPTION
**Pull request by tomap**
https://bip.sh/pricing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -709,7 +709,7 @@ Table of Contents
   * [000WebHost](https://www.000webhost.com/) — Zero cost website hosting with Apache, PHP, MySQL, cPanel with ads on the bottom of each pages!
   * [Alwaysdata](https://www.alwaysdata.com/) — 100 MB free web hosting with support for MySQL, PostgreSQL, CouchDB, MongoDB, PHP, Python, Ruby, Node.js, Elixir, Java, Deno, custom web servers, access via FTP, WebDAV and SSH; mailbox, mailing list and app installer included.
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
-  * [Bip](https://bip.sh) — Bip provides zero downtime deployment, a global CDN, SSL, and more for static websites. Free plan with 100GB/month bandwidth.
+  * [Bip](https://bip.sh) — Bip provides zero downtime deployment, a global CDN, SSL, and more for static websites. Free plan only for 7 days.
```
Comment (ripienaar) [inline README.md:712]: we can remove it then, it's not a free tier

Comment (abhay-ranawat) [reply README.md:712]: We should remove it

Comment (ripienaar) [reply README.md:712]: someone want to send a PR?

**Comment by tomap** [comment]
Following comments, I removed the lune

**Comment by ripienaar** [comment]
thanks!

**Pull request merged**